.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  goreleaser:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - name: Setup Snapcraft
+        run: |
+          sudo snap install snapcraft --classic
+          snapcraft login --with - <<<"${SNAP_TOKEN}"
+        env:
+          SNAP_TOKEN: ${{ secrets.SNAP_LOGIN }}
```

Comment:
@akram looks like this also needs to be added to the organization (we should coordinate offline)